x-pack/plugins/apm/public/components/app/ErrorGroupDetails/DetailView/ErrorTabs.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { isEmpty } from 'lodash';
+import { idx } from 'x-pack/plugins/apm/common/idx';
+import { APMError } from 'x-pack/plugins/apm/typings/es_schemas/Error';
+import {
+  getTabsFromObject,
+  Tab
+} from '../../../shared/PropertiesTable/tabConfig';
+
+export type ErrorTab = Tab | ExceptionTab | LogTab;
```

Comment:
I really like all of the Tab type stuff! I think it would be really helpful (to me at least) to refer to the "generic" tab as something like `PropertyTab` to help distinguish it from the others. Also `PropertyTabKey` for those special keys. What do you think?

Reply:
Good idea. Done!